x-pack/plugins/apm/server/routes/alerts/rule_types/get_group_action_variables.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { TRANSACTION_NAME } from '../../../../common/es_fields/apm';
+
+export const getGroupByActionVariables = (
+  groupByFields: Record<string, string>
+): Record<string, string> => {
+  return Object.keys(groupByFields).reduce<Record<string, string>>(
+    (acc, cur) => {
+      acc[cur === TRANSACTION_NAME ? 'transactionName' : cur] =
```

Comment:
This is for matching the naming of action variable `transactionName` that was already defined.

Reply:
> This is for matching the naming of action variable transactionName that was already defined.

Anything stopping us from changing it. It was introduced in this release cycle so it has not yet been released (won't be a breaking change)

Reply:
It would be easier to use `transaction.name` (without extra step of renaming it). Looking at the other variable names, such as serviceName, transasctionType, at the time,  it also seemed to be consistent to name variables like this.

I will change the action variable to be `transaction.name`.

Reply:
done